lib/rucio/daemons/transmogrifier/transmogrifier.py

```diff
@@ -477,7 +478,8 @@ def run_once(heartbeat_handler: "HeartbeatHandler", bulk: int, **_kwargs) -> boo
 
     worker_number, total_workers, logger = heartbeat_handler.live()
     stopwatch = Stopwatch()
-    blocklisted_rse_id = [rse["id"] for rse in list_rses({"availability_write": False})]
+    block_listed = {rse['rse']: rse["id"] for rse in list_rses({"availability_write": False})}
+    blocklisted_rse_id = list(block_listed.values())
```

Comment:
I’m having trouble understanding this change. Could you explain your reasoning?

Reply:
I need to have a list of RSEs (not RSE_ID) blocklisted for line 603

Reply:
I see, but I confess that I still find it confusing. It would have been easier to understand if named `blocklisted_rses` and `blocklisted_rse_ids` and if both were lists.

Reply:
I’m not convinced that the new variable names are superior to the ones I proposed.